src/allotropy/parsers/agilent_gen5/agilent_gen5_structure.py

```diff
@@ -337,6 +337,12 @@ def _get_measurement_labels(
                 f"{label_prefix}{excitation},{emission}"
                 for excitation, emission in zip(excitations, emissions, strict=True)
             ]
+            measurement_labels.extend(
+                [
+                    f"{label_prefix}{excitation.split('/')[0]},{emission.split('/')[0]}"
```

Comment:
Note for reviewer: The issue here was that we were not supporting the following format to create the measurement label for fluorescence detection point
![image](https://github.com/user-attachments/assets/aa3013e2-5f0f-4458-a1b8-55057deebd6e)